### ISSUE TITLE
Instructor Help Pages: replace magic numbers used as element IDs

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#8240]
For example, `src/main/webapp/partials/instructorHelpFAQ.jsp` has the code `<li id="faq2">`. The number `2` here is a _magic number_.  
Perhaps we can replace them with more meaningful words e.g. `<li id="faq-change-details-after-enrolling">` etc.?

**Comment by damithc** [comment]
To be done after #8190 to minimize conflicts

**Comment by ZiChuanBuXiu** [comment]
May I take this issue? I can subscribe #8190 and once it is done, I will try to fix this.

**Comment by damithc** [comment]
> May I take this issue? I can subscribe #8190 and once it is done, I will try to fix this.

@ZiChuanBuXiu Sure. Alternatively, you can submit a PR now and get it reviewed first, and resolve conflicts (if any) after #8190 is done.

**Comment by sukanta-27** [comment]
@ZiChuanBuXiu As you have already done one `d.FirstTimers` issue (#8161 ), I think (to be fair to other new contributors) you should now move on to other issues like `d.Contributors`, as you are no longer a first timer.

**Comment by dabbler011** [comment]
@damithc I suppose both @ZiChuanBuXiu and @sukanta-27 are done with their firstTimers. so, may I take this issue?

**Comment by damithc** [comment]
> @damithc I suppose both @ZiChuanBuXiu and @sukanta-27 are done with their firstTimers. so, may I take this issue?

Sure.
